packages/Graph/Microsoft.Bot.Component.MsGraph.csproj

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Same as the CalendarSkill.csproj - I think this can be removed

Reply:
Yep, stale - removed.